doc/source/data/overview.rst

```diff
@@ -15,53 +15,41 @@ Ray Data is a scalable data processing library for ML workloads, particularly su
 -  :ref:`Offline batch inference <batch_inference_overview>`
 -  :ref:`Data preprocessing and ingest for ML training <ml_ingest_overview>`
 
-It provides flexible and performant APIs for distributed data processing:
-
-- Simple transformations such as maps (:meth:`~ray.data.Dataset.map_batches`)
-- Global and grouped aggregations (:meth:`~ray.data.Dataset.groupby`)
-- Shuffling operations (:meth:`~ray.data.Dataset.random_shuffle`, :meth:`~ray.data.Dataset.sort`, :meth:`~ray.data.Dataset.repartition`).
+It provides flexible and performant APIs for distributed data processing. For more details, see :ref:`Transforming Data <transforming_data>`.
 
 Ray Data is built on top of Ray, so it scales effectively to large clusters and offers scheduling support for both CPU and GPU resources. Ray Data uses `streaming execution <https://www.anyscale.com/blog/streaming-distributed-execution-across-cpus-and-gpus>`__ to efficiently process large datasets.
 
-.. note::
-
-    Ray Data doesn't have a SQL interface and isn't meant as a replacement for generic
-    ETL pipelines like Spark.
-
 Why choose Ray Data?
 --------------------
 
 .. dropdown:: Faster and cheaper for modern deep learning applications
 
-    Ray Data is designed for deep learning applications that involve both CPU preprocessing and GPU inference. Through its powerful streaming :ref:`Dataset <dataset_concept>` primitive, Ray Data streams working data from CPU preprocessing tasks to GPU inferencing or training tasks, allowing you to utilize both sets of resources concurrently.
+    Ray Data is designed for deep learning applications that involve both CPU preprocessing and GPU inference. Through its powerful streaming execution, Ray Data streams working data from CPU preprocessing tasks to GPU inferencing or training tasks, allowing you to utilize both sets of resources concurrently.
```

Comment:
Feel like "Through its powerful streaming execution" is redundant since we say "Ray Data streams" in the next clause
```suggestion
    Ray Data is designed for deep learning applications that involve both CPU preprocessing and GPU inference. Ray Data streams working data from CPU preprocessing tasks to GPU inferencing or training tasks, allowing you to utilize both sets of resources concurrently.
```

Reply:
Updated.